src/utils/utils.ts

```diff
@@ -0,0 +1,10 @@
+import cryptoRandomString from 'crypto-random-string';
+// import { v4 as uuidv4 } from 'uuid';
+
+export const generateStandId = (): string => {
+  return cryptoRandomString({ length: 16, type: 'url-safe' });
```

Comment:
The stand reference ID can also be a UUID. Make the necessary changes in the schema so that you don't need to give this as a parameter when you're creating a stand.